src/Tribe/Editor/Warnings.php

```diff
@@ -46,6 +47,40 @@ public function show_recurring_event_warning_message( $post_id ) {
 		$this->render_notice( $warning );
 	}
 
+	/**
+	 * Add Provider missing warning for tickets.
+	 *
+	 * @since TBD
+	 */
+	public function add_commerce_provider_warning() {
+		$available_modules = array_diff_key( \Tribe__Tickets__Tickets::modules(), [ 'Tribe__Tickets__RSVP' => true ] );
+
+		if ( count( $available_modules ) > 0 ) {
+			return;
+		}
+
+		$this->render_notice( $this->get_commerce_provider_missing_warning_message() );
+	}
+
+	/**
+	 * Get the Commerce Provider missing warning message.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The Commerce Provider missing message.
+	 */
+	public function get_commerce_provider_missing_warning_message() {
+		$kb_url      = 'http://m.tri.be/1ao5';
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 6 spaces

<sub>hash: 00c9e53f0c179bf52e50a7080f21655e</sub>